app/src/main/res/values-pt/strings.xml

```diff
@@ -126,7 +126,7 @@
     <string name="performance_unblock_page">Desbloquear página</string>
     <string name="performance_title">Desempenho ruim do site</string>
     <string name="performance_message">Tente reduzir o tamanho da janela para melhorar o desempenho. Desbloquear esta página pode travar a sua app.</string>
-    <string name="voice_samples_collect_data_dialog_title">Permitir que %1$s colete amostras de voz para pesquisa\?</string>
+    <string name="voice_samples_collect_data_dialog_title">Permitir que %1$s colete amostras de voz para pesquisa?</string>
```

Comment:
s/colete/recolha/

colete is Brazilian Portuguese.

Reply:
@felipeerias I guess it's better to change it in Weblate first and the resubmit the PR?

Reply:
@pmatos @svillar Yes, the translation appears like that in Weblate, better to change it there and I will update the PR.

Reply:
I have fixed the string in Weblate and brought the change into this PR.

Reply:
lgtm